transformer_engine/pytorch/module/base.py

```diff
@@ -406,6 +406,23 @@ def __init__(self) -> None:
         self.fsdp_wrapped = False
         self.fsdp_group = None
         self._fp8_workspaces: Dict[str, Float8Tensor] = {}
+        self.activation_dtype: Optional[torch.dtype] = None
+
+        # Fast variant of __setattr__
+        # Note: torch.nn.Module has custom __setattr__ that handles
+        # modules, parameters, and buffers. This is unnecessary
+        # overhead when setting plain attrs.
+        self._fast_setattr: Callable[Tuple[str, Any], None]
```

Comment:
What is the purpose of this?

Reply:
I like to have a type hint to make usage clearer, but it's too long to fit in one line.